### PR TITLE
Issue #8: Allow Chrome flags to be specified by the client to allow launching in a containerized environment

### PR DESCRIPTION
**Pull request by joshmcarthur**
Adds a configuration hook to specify Chrome flags. These can be specified as an Array: `%w[headless no-sandbox]`, or as a string `--no-sandbox --headless`, or can be left blank.

This patch is necessary because running Chrome in a Docker container requires flags to be passed - specifically, the process must be run headlessly and with the no-sandbox option set. Because Lighthouse launches it's own instance of Chrome with it's own flags, we need to have the ability to customise how Chrome will be launched.

I expect that the Selenium webdriver options object can be passed to this config hook so that Lighthouse Matchers can inherit the Chrome flags that Selenium would use to launch the browser, leading to a consistent test environment.

### CODE REVIEW

File: lib/lighthouse/matchers.rb [hunk 1]
```diff
@@ -27,6 +28,13 @@ def runner
         @runner ||= proc { |cmd| `#{cmd}` }
       end
 
+      def chrome_flags
+        return unless @chrome_flags
```
Comment (G-Rath) [inline lib/lighthouse/matchers.rb]: Mainly for learning: 

What does this catch that `is_a?(Array)` won't?

Or is it just about strictly enforcing the return type to `Array | nil`?

Comment (joshmcarthur) [reply lib/lighthouse/matchers.rb]: We want to return `nil` if there is no value (as in it is falsey). This allows us to not add the option at all if the option is not specified.

Comment (G-Rath) [reply lib/lighthouse/matchers.rb]: Right, so it's the second one. thanks for confirming 👍

**Comment by Rabid-Dan** [comment]
probably worth adding a bit to the readme about the whole docker thing as well

**Comment by joshmcarthur** [comment]
@Rabid-Dan I have added documentation. @G-Rath are you happy to approve now?

**Pull request merged**